api/client/service/opts.go

```diff
@@ -491,6 +493,7 @@ func addServiceFlags(cmd *cobra.Command, opts *serviceOptions) {
 
 	flags.StringVarP(&opts.workdir, flagWorkdir, "w", "", "Working directory inside the container")
 	flags.StringVarP(&opts.user, flagUser, "u", "", "Username or UID (format: <name|uid>[:<group|gid>])")
+	flags.StringSliceVarP(&opts.group, flagGroupAdd, "g", []string{}, "User group or GID")
```

Comment:
-1 on the `-g`.
Could use some extra description. "Add additional groups to the container"